### ISSUE TITLE
reference to .jsprettifyrc is not working

### ISSUE THREAD

**Issue by dasois** [jonschlinkert/grunt-prettify#16]
My Gruntfile.js looks like this:
 prettify: {
            options: {
                prettifyrc: '.prettifyrc'
            }, ...
this is not working, a .prettifyrc is in the directory. If I copy the contents of the .prettifyrc and paste it to the Gruntfile.js it looks like this:
prettify: {
            options: {
                "indent": 4,
                "indent_char": " ",
                "condense": true,
                "indent_inner_html": true,
                "padcomments": true,
                "unformatted": [
                    "a",
                    "pre"
                ]
            }, ...

and this works!

**Comment by dasois** [comment]
maybe it's because there is also a .jsbeautify in the same dir..?

**Comment by jonschlinkert** [comment]
yeah, try getting rid of `.prettifyrc` and use the `.jsbeautify` instead. if that works I'll remove `.prettifyrc`

**Comment by Nagogus** [comment]
There was a mistake in examples. Change 'prettifyrc' property name to 'config'
